index.js

```diff
@@ -537,6 +565,16 @@
         other === this && 1 / other === 1 / this;
   }
 
+  //  Number$prototype$lte :: Number ~> Number -> Boolean
+  function Number$prototype$lte(other) {
+    return typeof other === 'object' ?
+      typeof this === 'object' &&
+      lte(this.valueOf(), other.valueOf()) :
+      isNaN(other) && isNaN(this) ||
+      typeof this !== 'object' &&
+      this <= other && (1 / this > 0 ? 1 : -1) <= (1 / other > 0 ? 1 : -1);
```

Comment:
Sorry. This expression was originally in another function and had an associated comment. The expression `(1 / a > 0 ? 1 : -1)` evaluates to the sign of `a`. It's used here to detect `-0`;